mlflow_faculty/mlflow_converters.py

```diff
@@ -128,3 +130,13 @@ def mlflow_timestamp_to_datetime_milliseconds(mlflow_timestamp):
 
 def mlflow_timestamp_to_datetime_seconds(mlflow_timestamp):
     return datetime.fromtimestamp(mlflow_timestamp, tz=UTC)
+
+
+def mlflow_viewtype_to_faculty_lifecycle_stage(mlflow_view_type):
+    if mlflow_view_type == ViewType.ACTIVE_ONLY:
+        return FacultyLifecycleStage.ACTIVE
+    elif mlflow_view_type == ViewType.DELETED_ONLY:
+        return FacultyLifecycleStage.DELETED
+    elif mlflow_view_type == ViewType.ALL:
+        return None
+    raise ValueError("Unexpected view_type: {}".format(mlflow_view_type))
```

Comment:
I feel like we would normally implement this as a dictionary?

Reply:
Makes sense